src/util/troll.hpp

```diff
@@ -36,5 +36,4 @@ namespace big::troll
 
 		g_pointers->m_gta.m_trigger_script_event(1, args, arg_count, -1, (int)eRemoteEvent::Bounty);
 	}
-
```

Comment:
Unnecessary whitespace change.

Reply:
The whitespace itself was unnecessary too. Ive developed minor ocd.

Reply:
You guys all hard at work making this menu very cool. Nice casino hack btw gir